build.gradle.kts

```
@@ -1,66 +1,77 @@
 import org.jetbrains.dokka.gradle.DokkaTask
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import java.net.URL
 
 plugins {
-    kotlin("jvm") version "1.7.20"
+    kotlin("multiplatform") version "1.7.20"
     kotlin("plugin.serialization") version "1.7.20"
     id("org.jetbrains.kotlinx.kover") version "0.6.1"
     id("org.jetbrains.dokka") version "1.7.20"
     `maven-publish`
-    `java-library`
     signing
 }
 
 group = "io.github.xn32"
 version = "0.2.1"
```

Let's start to prepare for version 0.3.0:
```suggestion
version = "0.3.0-SNAPSHOT"
```